src/components/carousel.js

```diff
@@ -0,0 +1,48 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import MovieCard from "./movie-card/movie-card";
+import SwiperCore, { Pagination, Navigation } from "swiper";
+SwiperCore.use([Pagination]);
+SwiperCore.use([Navigation]);
+
+export function Carousel(items) {
+  console.log(items);
+  return (
+    <>
+      <Swiper
+        navigation={true}
+        spaceBetween={30}
+        slidesPerView={5}
+        pagination={{}}
+        breakpoints={{
+          375: {
+            width: 375,
```

Comment:
This works, what I prefer doing is always assigning `width: 100%` to an element and limiting it's real width with `max-width: 375px`.

Since we're using images here there's a possibility of this going out of bounds on certain screen configurations, it might work perfectly fine.

I'll leave it up to you, both soolutions are probably fine here, these are just my recommendations.